### PR TITLE
New style

### PR DESCRIPTION
**Pull request by AadSchippers**
Amended files to support the new Pleio style

### CODE REVIEW

**Comment by bartjkdp** [comment]
Hi Aad, looks great. I see two small issues: 

- The letters "Pleio" of the logo supposed to be black
- I am missing the bounding box containing the login and password field.

image

image

The letters of Pleio supposed to be black.

**Comment by AadSchippers** [comment]
Fixed letter colouring.
Made the bounding box visible by altering the <fieldset> style attributes.

**Comment by bartjkdp** [comment]
Almost there ;) Please compare the CSS properties of .account__content carefully, some differences are there:

image

instead of the properties on Pleio.nl:

image

Also check the classes associated with the login button.

image

instead of the properties on Pleio.nl:

image

File: assets/less/pleio.less [hunk 1]
```diff
@@ -6,7 +6,12 @@
 fieldset {
     padding: 0;
     margin: 0;
-    border: 0;
+    border: 2;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    padding-left: 50px;
+    padding-right: 50px;
+
```
Comment (bartjkdp) [inline assets/less/pleio.less]: I think the above changes are not necessary. Please apply the styling to account__content.

File: core/templates/login.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
         </div>
 
         <div class="login__buttons">
-            <button name="login" class="button ___stretch ___active login__login" type="submit">{% trans "Login" %}</button>
+            <button name="login" class="button ___stretch ___outline ___active login__login" type="submit">{% trans "Login" %}</button>
```
Comment (bartjkdp) [inline core/templates/login.html]: I think the login button should not have ___outline.

File: core/templates/_logo.html [hunk 3]
```diff
@@ -1 +1 @@
...
....2 3.8 48 4 47.8 4.3 47.4 4.4 47 4.4 46.6 4.4 46.2 4.3 46 4 45.8 3.7 45.6 3.5 45.6 3.1L45.6 3.1ZM47.9 26.2L45.9 26.2 45.9 8.2 47.9 8.2 47.9 26.2 47.9 26.2Z" id="Shape"></path><path d="M52.2 17C52.2 15.7 52.4 14.5 52.8 13.4 53.2 12.3 53.7 11.3 54.4 10.5 55.1 9.7 55.9 9.1 56.9 8.6 57.9 8.1 59 7.9 60.2 7.9 61.4 7.9 62.5 8.1 63.5 8.6 64.5 9.1 65.3 9.7 66 10.5 66.7 11.3 67.2 12.3 67.6 13.4 68 14.5 68.2 15.7 68.2 17L68.2 17.5C68.2 18.8 68 20 67.6 21.1 67.2 22.2 66.7 23.2 66 24 65.3 24.8 64.5 25.4 63.5 25.9 62.5 26.3 61.4 26.6 60.2 26.6 59 26.6 57.9 26.4 56.9 25.9 55.9 25.5 55.1 24.8 54.4 24 53.7 23.2 53.2 22.2 52.8 21.1 52.4 20 52.2 18.8 52.2 17.5L52.2 17 52.2 17ZM54.2 17.6C54.2 18.6 54.3 19.5 54.6 20.4 54.9 21.3 55.3 22.1 55.8 22.7 56.3 23.4 56.9 23.9 57.7 24.3 58.5 24.7 59.3 24.9 60.3 24.9 61.3 24.9 62.1 24.7 62.9 24.3 63.6 23.9 64.3 23.4 64.8 22.7 65.3 22 65.7 21.3 66 20.4 66.3 19.5 66.4 18.6 66.4 17.6L66.4 17C66.4 16 66.3 15.1 66 14.2 65.7 13.3 65.3 12.5 64.8 11.9 64.3 11.2 63.7 10.7 62.9 10.3 62.1 9.9 61.3 9.7 60.3 9.7 59.3 9.7 58.5 9.9 57.7 10.3 57 10.7 56.3 11.2 55.8 11.9 55.3 12.6 54.9 13.4 54.6 14.2 54.3 15.1 54.2 16 54.2 17L54.2 17.6 54.2 17.6Z" id="Shape"></path></g></g></svg>
```
Comment (bartjkdp) [inline core/templates/_logo.html]: Great that's it 👍

**Comment by AadSchippers** [comment]
Thanks for your comment. I believe it's sorted now. ;)

**Pull request merged**